kura/examples/org.eclipse.kura.example.ble.tisensortag.tinyb/src/main/java/org/eclipse/kura/example/ble/tisensortag/tinyb/TiSensorTag.java

```diff
@@ -299,7 +299,11 @@ public void enableAccelerometer(byte[] config) {
             try {
                 accEnableChar = this.gattServices.get(MOVEMENT)
                         .findCharacteristic(TiSensorTagGatt.UUID_MOV_SENSOR_ENABLE);
-                accEnableChar.writeValue(config);
+                byte[] oldConfig = accEnableChar.readValue();
```

Comment:
Maybe this code can be extracted into a separate method, to reduce code duplication